code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm

```diff
@@ -291,17 +291,17 @@
 /datum/crafting_recipe/arrow
 	name = "Arrow"
 	result = /obj/item/ammo_casing/caseless/arrow
-	time = 40
+	time = 20
 	reqs = list(/obj/item/stack/sheet/mineral/wood = 1,
-				 /obj/item/weaponcrafting/silkstring = 1,
```

Comment:
They should need to tie the arrow head with something like a strong piece of woven cloth, not just any piece.